### PR TITLE
added Screen Sharing Page

### PR DESCRIPTION
**Pull request by ctrlaltf24**
https://github.com/QubesOS/qubes-issues/issues/6426

### CODE REVIEW

File: docs/configuration/screen-share.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+4. Open applications `DISPLAY=:10 xterm` where xterm can be any binary on your system
+
+## Qubes Connect TCP Service
+
+These steps are a simpler version of [The Qubes Firewall](https://www.qubes-os.org/doc/firewall/#opening-a-single-tcp-port-to-other-network-isolated-qube). In dom0 execute these steps:
+
+1. Edit /etc/qubes-rpc/policy/qubes.ConnectTCP 
+
+2. Add the line: `PRESENTATION-QUBE @default ask,target=CONTENT-QUBE`
```
Comment (deeplow) [inline docs/configuration/screen-share.md:31]: Maybe replace:
 -  `PRESENTATION-QUBE` with `<PRESENTATION_QUBE>` 
 -  `CONTENT-QUBE` with <CONTENT_QUBE>`

to be consistent with the official docs. Also in other places of this document.

File: docs/configuration/screen-share.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Notes
+
+- To reset the VNC password delete `/home/user/.vnc/passwd` in the Content Qube
+
+- The Content Qube does not need to have access to the internet
+
+- Opening up a TCP port between Qubes Should only be used as a last resort, use sparingly.
+
+- After you are done, remove the line you added in `/etc/qubes-rpc/policy/qubes.ConnectTCP`
```
Comment (deeplow) [inline docs/configuration/screen-share.md:65]: I think this disclaimer should be at the beginning. Maybe something like:

>  :warning: **Warning**:
> This guide involves opening up a TCP port between qubes. This is discouraged from the security standpoint and should only be used as a last resort, use sparingly. After use, please remove the qubes RPC (see the notes at the bottom)

**Comment by awokd** [comment]
@ctrlaltf24 can you do the edits deeplow suggested?

**Comment by ctrlaltf24** [comment]
There we go, added x11vnc section for sharing existing screen. I am aware the misc updates commit isn't signed (forgot -S), but its still me.

**Pull request merged**